qir/stdlib/src/stdlib.c

```diff
@@ -2,7 +2,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
-
+/*
```

Comment:
It looks like these updates to the stdlib may have snuck in here inadvertently. Are these related to the grouping pass? If not, it would be better to not include them here and leave those for a separate PR.

Reply:
Yes, this should indeed just be the QIR from main.